Kernel/Devices/Storage/SD/Registers.h

```diff
@@ -256,6 +256,14 @@ static_assert(AssertSize<CardIdentificationRegister, 16>());
 union CardSpecificDataRegister {
     u64 raw[2];
 
+    struct [[gnu::packed]] {
+        u32 : 32;
+        u32 : 32;
+        u32 : 32;
+        u32 : 22;
+        u32 csd_structure : 2;
+    };
```

Comment:
Isn't this only 24 bits in the last section?
(Phone review so did not check the other versions)